test/check-application

```diff
@@ -1131,6 +1193,58 @@ class TestFiles(testlib.MachineCase):
         b.click("button.pf-m-link")
         b.wait_not_present(".pf-v5-c-modal-box")
 
+        # Test permissions in details view
+        b.mouse("[data-item='home']", "dblclick")
+        b.mouse("[data-item='admin']", "dblclick")
+        b.click("button[aria-label='Display as a list']")
+        basedir = "/home/admin"
+        for i in range(8):
+            m.execute(f"runuser -u admin touch {basedir}/file{i}")
+
+        def check_perms_match(file: str, basedir: str) -> None:
+            ls = m.execute(f"ls -l {basedir}/{file}")[:10]
+            ui = b.text(f"[data-item='{file}'] td:nth-child(4) pre").replace(" ", "")
+            self.assertEqual(ls[1:], ui)
+
+        # Simple permissions
+        m.execute(f"chmod 000 {basedir}/file0")
+        m.execute(f"chmod 111 {basedir}/file1")
+        m.execute(f"chmod 222 {basedir}/file2")
+        m.execute(f"chmod 333 {basedir}/file3")
+        m.execute(f"chmod 444 {basedir}/file4")
+        m.execute(f"chmod 555 {basedir}/file5")
+        m.execute(f"chmod 666 {basedir}/file6")
+        m.execute(f"chmod 777 {basedir}/file7")
+
+        for i in range(8):
+            check_perms_match(f"file{i}", basedir)
+
+        # Test different permissions for owner/group/others
+        m.execute(f"chmod 411 {basedir}/file0")
+        m.execute(f"chmod 546 {basedir}/file1")
+        m.execute(f"chmod 337 {basedir}/file2")
+        m.execute(f"chmod 755 {basedir}/file3")
+        m.execute(f"chmod 613 {basedir}/file4")
+        m.execute(f"chmod 711 {basedir}/file5")
+        m.execute(f"chmod 531 {basedir}/file6")
+        m.execute(f"chmod 740 {basedir}/file7")
+
+        for i in range(8):
+            check_perms_match(f"file{i}", basedir)
+
+        # Test permissions with setuid, setgid, sticky
+        m.execute(f"chmod 1000 {basedir}/file0")
+        m.execute(f"chmod 2111 {basedir}/file1")
+        m.execute(f"chmod 3222 {basedir}/file2")
+        m.execute(f"chmod 4333 {basedir}/file3")
+        m.execute(f"chmod 5444 {basedir}/file4")
+        m.execute(f"chmod 6555 {basedir}/file5")
+        m.execute(f"chmod 7666 {basedir}/file6")
+        m.execute(f"chmod 3777 {basedir}/file7")
+
+        for i in range(8):
+            check_perms_match(f"file{i}", basedir)
```

Comment:
The tooltip is not tested, this would be nice, maybe even with a pixel test? This can be done quite easily:

```
1004   │         b.mouse("#current-memory-usage", "mouseenter")
1005   │         b.wait_in_text(".pf-v5-c-tooltip", "B total")
1006   │         b.mouse("#current-memory-usage", "mouseleave")
```

Don't have to loop to check, can just be a selective one.